code/modules/food_and_drinks/food/snacks.dm

```diff
@@ -212,7 +212,7 @@ All foods are distributed among various categories. Use common sense.
 	if(bonus_reagents && bonus_reagents.len)
 		for(var/r_id in bonus_reagents)
 			var/amount = bonus_reagents[r_id]
-			if(r_id == /datum/reagent/consumable/nutriment || r_id == /datum/reagent/consumable/nutriment/vitamin)
+			if(r_id == /datum/reagent/consumable/nutriment || r_id == /datum/reagent/consumable/nutriment/vitamin || r_id == /datum/reagent/consumable/nutriment/protein)
```

Comment:
Could you improve this to check for a variable in the type, with `initial()` ?

Reply:
sounds a tad out of scope for this already huge pr if you ask me

Reply:
Fair